### ISSUE TITLE
Compiled version

### ISSUE THREAD

**Issue by Dante383** [Knuk/MinecraftBot#1]
Hi, I would like to test it but I can not compile. Could I ask for spreading the .exe file? Thank you very much in advance

**Comment by samuellapointe** [comment]
Hello, try this: https://www.dropbox.com/s/3zh4txqru9fx3od/Bot.zip

It's the latest version, I haven't tested it a lot yet so it might not be as stable!

**Comment by Dante383** [comment]
Thank you very much, if I find it I will show any errors. I have another question, can somehow do not use password? (no-premium)

**Comment by samuellapointe** [comment]
Yup, if you're connecting to a server where online-mode: false in server.properties, then you can enter anything in the username and password boxes.

**Comment by Dante383** [comment]
22:00:55 [INFO] /127.0.0.1:53152 lost connection
Hmm, online-mode is false. Something i did wrong?

**Comment by samuellapointe** [comment]
Is the server running any plugins? I have trouble connecting to servers that aren't vanilla

**Comment by Dante383** [comment]
Yup, server is bukkit. I try vanilla, give me 10 minutes.

**Comment by Dante383** [comment]
if the server version is 1.5.2, the bot work?

**Comment by samuellapointe** [comment]
Oh no it's running on 1.8, I haven't tried making it work on previous versions, sorry

**Comment by Dante383** [comment]
Oh. So i must download 1.8, no problem.

**Comment by Dante383** [comment]
Only 1.8, 1.8.3 no? sorry to be flooded with questions you D:

**Comment by samuellapointe** [comment]
Don't worry about the questions! It works on any 1.8.x version so far, so it should work with 1.8.3

**Comment by RealRedcodi** [comment]
Hey, can someone compile it for MAC Please?
